src/screens/perks/children/questsCard.tsx

```
@@ -0,0 +1,109 @@
+const ICONS: Record<string, string> = {
+  'check-circle': 'progress-check',
+  pencil: 'pencil-outline',
+  comment: 'comment-outline',
+  'chevron-up-circle': 'chevron-up-circle-outline',
+  repeat: 'repeat',
+};
+
+const TIERS = ['daily', 'weekly', 'monthly'] as const;
+
+const byId = (arr?: { id: string }[]) =>
+  Object.fromEntries((arr || []).map((q) => [q.id, q]));
```

ESLint

[error] Delete `⏎·`

(prettier/prettier)